app/views/manuals/_content_for_manual.html.erb

```diff
@@ -0,0 +1,7 @@
+<% content_for :extra_headers do %>
+  <% if presented_manual.summary.present? %>
+    <meta name='description' content='<%= presented_manual.summary %>' />
+  <% end %>
```

Comment:
I wonder if the arguments to this should be `title` and `description` rather than `title` and `presented_manual`.  That way we could re-use it in the updates view.